model/DeliveryContainerService.php

```diff
@@ -177,4 +169,44 @@ protected function getActiveFeatures(\core_kernel_classes_Resource $delivery)
             $delivery->getOnePropertyValue($this->getProperty(self::TEST_RUNNER_FEATURES_PROPERTY))
         );
     }
+
+    /**
+     * @param core_kernel_classes_Resource $delivery
+     * @return array
+     */
+    protected function getPluginsDisabledForDelivery(core_kernel_classes_Resource $delivery)
+    {
+        $enabledPlugins = [];
+        $disabledPlugins = [];
+        try {
+            $testRunnerFeatureService = $this->getServiceLocator()->get(TestRunnerFeatureService::SERVICE_ID);
+            $allTestRunnerFeatures = $testRunnerFeatureService->getAll();
+
+            if (count($allTestRunnerFeatures) === 0) {
+                return $disabledPlugins;
+            }
+
+            $activeTestRunnerFeaturesIds = $this->getActiveFeatures($delivery);
+            foreach ($allTestRunnerFeatures as $feature) {
+                if (in_array($feature->getId(), $activeTestRunnerFeaturesIds)) {
```

Comment:
Nip: it will be good to use strict mode for `in_array` function

Reply:
Fixed